docs/scenarios_authoring/BasicNodes.md

```diff
@@ -4,77 +4,75 @@ sidebar_position: 2
 
 # Basic Components
 
-Nodes work with a data records. They can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
+Nodes work with data records. They can produce, fetch, send and collect data or organize the flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
```

Comment:
maybe it is better to write "data flow" here, to be more precise, even if this means we use word data twice

Reply:
"Nodes can manipulate the data flow by producing, fetching, sending or collecting data"?

Reply:
You can write "Nodes can manipulate the data flow by producing, fetching, sending or collecting data", or it can stay as is